Description: Fix typos in weather.pl.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-08-24

--- a/Src/weather.pl
+++ b/Src/weather.pl
@@ -114,7 +114,7 @@
 $line   .= $newLine;
 chomp $line;
 
-$line = ~s/<hr>//;
+$line =~ s/<hr>//;
 $line = "METAR $station $line";
 
 close DATA;
@@ -178,7 +178,7 @@
     $obsc	= "";
     $misc	= "";
     $j		= 0;
-    $curent	= $args[$i]
+    $curent	= $args[$i];
     $wasCloud   = "no";
 
     debug("cur : $curent");
